Gemfile.lock

```diff
@@ -370,6 +370,3 @@ DEPENDENCIES
   uglifier (>= 1.0.3)
   will_paginate (>= 3.0.6)
   will_paginate-bootstrap (>= 1.0.1)
-
```

Comment:
Hi, this is looking good -- do you think you could revert this file as it's unrelated to your changes? Sorry, this happens sometimes. Thanks!